dora_node_hub/dora_gemini_diff_drive_navigation/README.md

```diff
@@ -0,0 +1,90 @@
+# dora_gemini_diff_drive_navigation
+
+Relies on Google's LLM model: [Gemini](https://gemini.google.com/) for navigating a differential drive robot.
+
+## Getting started
+
+- Install it with uv:
+
+```bash
+uv venv -p 3.11 --seed
+uv pip install -e .
+```
+
+## Contribution Guide
+
+- Format with [ruff](https://docs.astral.sh/ruff/):
+
+```bash
+uv pip install ruff
+uv run ruff check . --fix
+```
+
+- Lint with ruff:
+
+```bash
+uv run ruff check .
+```
+
+- Test with [pytest](https://github.com/pytest-dev/pytest)
+
+```bash
+uv pip install pytest
+uv run pytest . # Test
+```
+
+## YAML Specification
+
+### inputs
+  - `tick`: Rate at which the control loop is attempted to be run.
+  - `cmd_vel_tick`: Rate at which the command velocity should be published.
+  - `image`: Camera image to be used as input for the prompt.
+  - `command`: Command to be prompted to Gemini's API.
+
+### outputs
+  - `cmd_vel`: Velocity command to be forwarded to a differential drive controller.
+
+### envs
+  - `COMMAND`: Optional static command. You can use always this command if no *command* is set as *input*. If both (*COMMAND* env and *command* input) are set, this would act as initial command and then be replaced by the input.
```

Comment:
@francocipollone 
```suggestion
  - `COMMAND`: Optional static command. You can use this command if no *command* is set as *input*. If both (*COMMAND* env and *command* input) are set, this would act as initial command and then be replaced by the input.
```